.github/workflows/e2e_detox_mapping.yml

```diff
@@ -141,7 +141,9 @@ jobs:
             select(contains(["Pagination"])) += ["Appointments", "Claims", "Appeals", "AppealsExpanded", "Prescriptions", "Messages", "Payments", "VaccinesScreen"] |
             select(contains(["SnackBar"])) += ["Claims", "Messages", "ContactInformation", "PersonalInformationScreen", "DirectDeposit"] |
             select(contains(["VABulletList"])) += ["Appeals", "AppealsExpanded", "Claims", "Prescriptions", "Cerner", "Messages", "Onboarding", "SignIn", "SettingsScreen"] |
-            select(contains(["CrisisLineCta"])) += ["HomeScreen", "SignIn", "LoginScreen", "VeteransCrisisLine"] |
+            select(contains(["CrisisLineButton"])) += ["HomeScreen", "SignIn", "LoginScreen", "VeteransCrisisLine"] |
+            select(contains(["ActivityButton.tsx"])) += ["HomeScreen", "Prescriptions", "Appointments", "AppointmentsExpanded", "Claims", "Appeals", "AppealsExpanded", "DecisionLetters"] |
```

Comment:
Would remove the expanded ones (appeals and appointments), they wouldn't add anything for changes to the activity buttons